packages/orchestrator-ui-components/src/components/WfoForms/formFields/SelectField/SelectField.tsx

```diff
@@ -98,7 +98,9 @@ export function UnconnectedSelectField({
         useWithOrchestratorTheme(getSelectFieldStyles);
 
     useEffect(() => {
-        onChange(selectedValue?.value);
+        if (selectedValue && selectedValue.value !== 'undefined') {
```

Comment:
`selectedValue && selectedValue.value` can be replaced with `selectedValue?.value`. And the question mark in the onChange can be removed

Reply:
Thanks, but if I do this I get errors on runtime, not sure how to fix it properly in the FE